tests/e2e/e2e_test.go

```diff
@@ -319,6 +319,57 @@ func (s *IntegrationTestSuite) TestConcentratedLiquidity() {
 
 }
 
+func (s *IntegrationTestSuite) TestStableSwapPostUpgrade() {
+	if s.skipUpgrade {
+		s.T().Skip("Skipping StableSwapPostUpgrade test")
+	}
+
+	chainA := s.configurer.GetChainConfig(0)
+	chainANode, err := chainA.GetDefaultNode()
+	s.Require().NoError(err)
+
+	const (
+		poolFile   = "stablePool.json"
+		walletName = "stable-swap-wallet"
+
+		denomA = "stake"
+		denomB = "uosmo"
+
+		minAmountOut = "1"
+
+		epochIdentifier = "day"
+	)
+
+	coinAIn, coinBIn := fmt.Sprintf("20000%s", denomA), fmt.Sprintf("1%s", denomB)
+
+	poolId := chainANode.CreateStableswapPool(poolFile, initialization.ValidatorWalletName)
```

Comment:
Here, a new pool is created but we want to swap against the pool that was created pre-upgrade